cmd/auth.go

```diff
@@ -244,15 +230,15 @@ func Cancel(username string, password string, yes bool) error {
 	err = auth.Delete(c, username)
 	cleanup := fmt.Errorf("\n%s %s\n\n", "409", "Conflict")
 	if reflect.DeepEqual(err, cleanup) {
```

Comment:
We need better errors all around the SDK! I referenced this specific code earlier in my better errors issue deis/controller-sdk-go#7